cloud-canary.py

```diff
@@ -99,136 +100,154 @@ def main():
     return args
 
 
-def deploy_instance(args):
-    api_key = args['acskey']
-    secret_key = args['acssecret']
-    zonename = args['zonename']
-    endpoint = "https://" + args['endpoint'] + "/compute"
-    template = args['template']
-    offering = args['offering']
+def create_network(cs, name, zoneid):
+    """Creates a managed private network"""
 
-    logging.basicConfig(
-        format='%(asctime)s %(pathname)s %(levelname)s:%(message)s',
-        level=logging.DEBUG,
-        filename=logfile)
-    logging.getLogger().addHandler(logging.StreamHandler())
+    noid = cs.listNetworkOfferings()['networkoffering'][0]['id']
+    return cs.createNetwork(
+            displaytext="cloud-canary test network",
+            name=name, networkofferingid=noid,
```

Comment:
```suggestion
            name=name,
```

`networkofferingid` isn't needed anymore so you don't even have to fetch the privnet offering.